pyproject.toml

```diff
@@ -42,6 +42,8 @@ pyannote-audio = "^3.1.1"
 pydantic = ">=2.0"
 pydantic-settings = "^2.1.0"
 python-multipart = "^0.0.9"
+psycopg = {extras = ["binary"], version = "^3.2.1"}
+psycopg2-binary = "^2.9.9"
```

Comment:
Do we really need both of them? 
On [psycopg (3.2.1)](https://pypi.org/project/psycopg/) page it says:
> Despite the lack of number in the package name, this package is the successor of [psycopg2](https://pypi.org/project/psycopg2/).
> Please use the psycopg2 package if you are maintaining an existing program using psycopg2 as a dependency. If you are developing something new, Psycopg 3 is the most current implementation of the adapter.

Reply:
We need psycopg2-binary for the postgresql. But we don't really use psycopg.
Let me check if we can remove the psycopg.

Reply:
Upddated the sqlurl to use the psycopg3